vg/vgeps/vgeps.go

```diff
@@ -17,8 +17,15 @@ import (
 	"time"
 
 	"gonum.org/v1/plot/vg"
+	"gonum.org/v1/plot/vg/draw"
 )
 
+func init() {
+	draw.RegisterFormat("eps", func(w, h vg.Length) vg.CanvasWriterTo {
+		return New(w, h)
+	})
```

Comment:
One line? (consistency with vgpdf - or make that 3 lines). Given the simplicity of the values being returned, I think one line is better).

Reply:
in the spirit of keeping the col-count around 80, I changed it to the 3-line form though out.